src/components/AssetDetailsPanel/AssetDetailsPanelPure.tsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { WithAssetDataProps } from '../../hoc/withAsset';
 import { DescriptionList } from '../DescriptionList';
+import { MetaDescriptionListProps } from '../DescriptionList';
+
+export type MetaAssetDetailsProps = MetaDescriptionListProps;
```

Comment:
Why do you do this? `MetaDescriptionListProps` also exported

Reply:
will fix